client/src/js/samples/components/LabelEditor/LabelItem.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { Label, Icon } from "../../../base";
+
+const getInitialState = ({ name, color, description, id }) => ({
+    name: name || "",
+    color: color || "",
+    description: description || "",
+    id: id || ""
+});
+
+export const getContrastColor = props => {
+    const red = parseInt(props.color.substr(1, 2), 16);
+    const green = parseInt(props.color.substr(3, 2), 16);
+    const blue = parseInt(props.color.substr(5, 2), 16);
+    const yiq = (red * 299 + green * 587 + blue * 114) / 1000;
+    return yiq >= 128 ? "black" : "white";
+};
+
+export const StyledLabel = styled(Label)`
```

Comment:
`Styled` prefix for components is only used when it is the top-level component in another component. This should be renamed. Something like `LabelItemExample`.